core/server/master/src/test/java/alluxio/master/file/loadmanager/LoadManagerTest.java

```diff
@@ -233,4 +233,45 @@ else if (status.stream().allMatch(s -> s.getCode() != Status.OK.getCode().value(
     assertTrue(loadJob.getTotalBlockCount() > 5000);
     assertTrue(loadManager.submitLoad(new LoadJob("test", 1000)));
   }
+
+  @Test
+  public void testSchedulingFullCapacity() throws Exception {
+    FileSystemMaster fileSystemMaster = mock(FileSystemMaster.class);
+    FileSystemContext fileSystemContext = mock(FileSystemContext.class);
+    CloseableResource<BlockWorkerClient> blockWorkerClientResource = mock(CloseableResource.class);
+    BlockWorkerClient blockWorkerClient = mock(BlockWorkerClient.class);
+    ImmutableList.Builder<WorkerInfo> workerInfos = ImmutableList.builder();
+    for (int i = 0; i < 1000; i++) {
+      workerInfos.add(new WorkerInfo().setId(i).setAddress(
+          new WorkerNetAddress().setHost("worker" + i).setRpcPort(1234)));
+    }
+    when(fileSystemMaster.getWorkerInfoList())
+        .thenReturn(workerInfos.build());
+    List<FileInfo> fileInfos = generateRandomFileInfo(2000, 50, 64 * 1024 * 1024);
```

Comment:
I would use 64 * Constants.MB for file size

Reply:
+1

Reply:
I remember there were some constant but i couldn't find it, thanks!